task-json/fetch.js

```diff
@@ -0,0 +1,22 @@
+global.fetch = require("cross-fetch");
+
+
+function readJson () {
+    fetch('https://gist.githubusercontent.com/keeguon/2310008/raw/bdc2ce1c1e3f28f9cab5b4393c7549f38361be4e/countries.json')
+    .then(response => {
+        if (!response.ok) {

```

Comment:
response.ok could have values other than 200, it could be accepted here, but in real live scenarios it is not